ui/apps/platform/src/Containers/Integrations/IntegrationForm/Forms/EmailIntegrationForm.tsx

```diff
@@ -455,6 +457,18 @@ function EmailIntegrationForm({
                             ))}
                         </SelectSingle>
                     </FormLabelGroup>
+                    <FormLabelGroup label="" fieldId="notifier.email.disableTLSCertificateValidation" errors={errors}>
+                        <Checkbox
+                            label="Disable TLS certificate validation"
+                            id="notifier.email.disableTLSCertificateValidation"
+                            isChecked={values.notifier.email.disableTLSCertificateValidation}
+                            onChange={(event, value) =>
+                                updateStartTLSAuthMethodOnChange(value, event)
+                            }
+                            onBlur={handleBlur}
+                            isDisabled={!isEditable || values.notifier.email.startTLSAuthMethod == 'DISABLED'}
+                        />
+                    </FormLabelGroup>
```

Comment:
Hmmm, I don't remember (maybe never knew) why there are two versions of this form.

Sounds like we need more guidance from the @pedrottimark or @sachaudh

Reply:
@vjwilson **Configure delivery destinations** in **Vulnerability Reporting** has **Create email notifier**
![EmailNOtifierForm](https://github.com/user-attachments/assets/dbb4fdd2-df33-47de-98b8-7948e82a2141)